app/modules/encounters/resources.py

```diff
@@ -197,8 +197,10 @@ def delete(self, encounter):
         resp = make_response()
         resp.status_code = 204
         sighting_id = None
+        deleted_individuals = None
         if result is not None:
             sighting_id = result.get('deletedSighting', None)
+            deleted_individuals = result.get('deletedIndividuals', None)
```

Comment:
If you change the default here to an empty list [] then you don't need the if deleted_individuals check below